sycl/test-e2e/Basic/kernel_info.cpp

```diff
@@ -100,6 +100,10 @@ int main() {
   const cl_uint compileNumSg =
       krn.get_info<info::kernel_device_specific::compile_num_sub_groups>(dev);
   assert(compileNumSg <= maxNumSg);
+  const size_t spillMemSz =
+      krn.get_info<ext::intel::info::kernel_device_specific::spill_memory_size>(
+          dev);
+  assert(spillMemSz >= 0);
```

Comment:
This should be hidden under an `if`, the query is not guaranteed to be supported everywhere. The same goes for other change in this file.